### PR TITLE
Add deprecation warning on st2ctl start if st2/python is python 2.x

### PR DESCRIPTION
**Pull request by amanda11**
Add warning messages in yellow to screen if perform st2ctl start (or restart) and /opt/stackstorm/st2/bin/python is python 2.x

Addresses part of #5041 and #4938

### CODE REVIEW

**Comment by amanda11** [comment]
image

File: st2common/bin/st2ctl [hunk 1]
```diff
@@ -87,6 +87,11 @@ function st2start() {
   for COM in ${COMPONENTS}; do
     service_manager ${COM} start
   done
+  PYTHON_VERSION=$(/opt/stackstorm/st2/bin/python --version 2>&1 | awk '{print $2'} | awk -F"." '{print $1}')
+  if [ "${PYTHON_VERSION}" = "2" ]; then
+    echo -e "\e[33mDeprecation warning: Support for python 2 will be removed in future StackStorm releases. Please ensure that all packs used are python 3 compatible. Your StackStorm installation may be upgraded from python 2 to python 3 in future platform releases. It is recommended to plan the manual migration to a python 3 native platform, e.g. Ubuntu 18.04 LTS or CentOS/RHEL 8. \e[0m\n"
```
Comment (armab) [inline st2common/bin/st2ctl:92]: That looks nice! 

Can we include the Deprecation warning for the st2ctl reload/register actions as well to make it even more visible people can't ignore?

Comment (amanda11) [reply st2common/bin/st2ctl:92]: That should be easy enough to add - will do.

Comment (nzlosh) [reply st2common/bin/st2ctl:92]: Nothing major, but the second awk is redundant. A single call to `awk` can give the python major version, as follows

```
$ python --version 2>&1 | awk -F"[. ]" '{print $2}'
3
```

Comment (amanda11) [reply st2common/bin/st2ctl:92]: Cool - will fix that up as well. And also try and remember the bit of ST2 I stole it from!

Comment (amanda11) [reply st2common/bin/st2ctl:89]: @armab Added warning into st2ctl reload and st2ctl restart-component. (start and restart covered by logic already there). 
@nzlosh Thanks for the awk comment, much neater now, that's much neater (I'd stolen it from st2-self-check - but I won't patch that up under this PR).

File: CHANGELOG.rst [hunk 2]
```diff
@@ -27,6 +27,9 @@ Added
   Python 2. (new feature) #5043
 
   Contributed by @amanda11
+* Added deprecation warning to st2ctl start,  if st2 python version is Python 2. (new feature) #5044
```
Comment (armab) [inline CHANGELOG.rst:30]:
```suggestion
* Added deprecation warning to st2ctl, if st2 python version is Python 2. (new feature) #5044
```

**Review by armab — APPROVED** [review]
Looks good to me! :+1: 

Thanks @nzlosh for additional awk tips!

**Pull request merged**